miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -209,6 +209,13 @@
   jQuery(document).ready(function () {
     // Attach Parsley form validator
     Validate.attachParsley('#pool-form');
+
+    // store duplicate indices sequences
+    Pool.duplicateIndices = ${pool.hasDuplicateIndices()};
```

Comment:
The `config` object is restricted to the `list` and `bulk` sections, and the edit Pool page isn't a member of either of those. What do you think of instead namespacing it within `Pool.indices`?